frontend/src/App.js

```diff
@@ -15,9 +15,10 @@ function App() {
       <main className="py-3">
           <Container>
             <Routes>
-                <Route path='/' element={<HomeScreen />} exact />
+                <Route path='/' element={<BrowsingScreen />} exact />
+                <Route path="/browse" element={<BrowsingScreen />} exact></Route>
                 <Route path='user-profile' element={<UserProfileScreen />} exact />
-                <Route path='sign-in' element={<SignIn />}/>
+                <Route path='login' element={<SignIn />}/>
```

Comment:
```suggestion
                <Route path='sign-in' element={<SignIn />}/>
```
for consistency, should be sign-in instead of login